kensu/google/cloud/bigquery/job/bigquery_stats.py

```diff
@@ -17,16 +19,26 @@ def compute_bigquery_stats(table_ref=None, table=None, client=None, stats_aggs=N
         logger.debug('Got empty statistic listing from remote service, proceeding with fallback statistic list')
         stats_aggs = generate_fallback_stats_queries(table)
 
-    selector = ",".join([sql_aggregation + " " + col + "_" + stat_name
+    list_of_unnested = [k.name for k in table.schema if k.fields!=() and k.is_nullable==False]
+
+    selector = ",".join([sql_aggregation + " " + col.replace(".","__ksu__") + "_" + stat_name
```

Comment:
Maybe one or two lines of comments for others to understand the hack, along those lines: 
"BigQuery nested and repeated columns in schemas are not supported in SparkSQL, as a workaround we flatten the schema and replace the infix dots with a "__ksu__" string. once the SparkSQL parser has done is job we convert back to the dotted notation for displaying stats"

Reply:
ok added, that was not the reason, it is because Bigquery doesn't support `.` in schema fields